core/src/main/java/org/mule/DefaultMessageCollection.java

```diff
@@ -278,15 +269,11 @@ public int size()
         }
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
     public String getPayloadAsString(String encoding) throws Exception
```

Comment:
Same comments for all the methods that are still in the code but have no @Override anymore